### ISSUE TITLE
Error when setting compiler settings

### ISSUE THREAD

**Issue by lsschmid** [cda-tum/mqt-bench#224]
I get an error whenever I try to set the compiler settings for the qiskit compiler.

minimal  example
```
from mqt.bench import get_benchmark, CompilerSettings
compiler_settings = CompilerSettings(
    qiskit={"optimization_level": 1},
)
get_benchmark(benchmark_name="ghz",
                level=2,
                circuit_size=3,
                compiler="qiskit",
                compiler_settings=compiler_settings,)
```

Error:
```
    376 if compiler == "qiskit":
    377     assert compiler_settings.qiskit is not None
--> 378     opt_level = compiler_settings.qiskit.optimization_level
    379     return qiskit_helper.get_native_gates_level(qc, gate_set_name, circuit_size, opt_level, False, True)
    380 if compiler == "tket":

AttributeError: 'dict' object has no attribute 'optimization_level'
```

I use the current version 1.0.0.

should probably be fixed by changing to "...qiskit["optimization_level"] ;)

**Comment by nquetschlich** [comment]
Thanks a lot for reporting. Actually, the issue here is the deprecated documentation in the readme. I fixed it in #225, it should be used like this:

```python
from mqt.bench import get_benchmark, CompilerSettings, QiskitSettings
compiler_settings = CompilerSettings(qiskit=QiskitSettings(optimization_level=1),)
get_benchmark(benchmark_name="ghz",
                level=2,
                circuit_size=3,
                compiler="qiskit",
                compiler_settings=compiler_settings,)
```

**Comment by lsschmid** [comment]
Fixed the problem, thanks! 🙏
